src/resources/setup_intent_ext.rs

```diff
@@ -32,6 +39,17 @@ impl SetupIntent {
     ) -> Response<SetupIntent> {
         client.post_form(&format!("/setup_intents/{}/confirm", setup_id), &params)
     }
+
```

Comment:
```suggestion

```
I think `fmt` is removing this extra line causing the CI issue. Running `cargo make fmt` should fix it